test/SlowTests/Server/Documents/ETL/Queue/AzureQueueStorageEtlTests.cs

```diff
@@ -0,0 +1,465 @@
+                connectionString: ConnectionString);
+
+            var exportFile = GetTempFileName();
+
+            var exportOperation = await srcStore.Smuggler.ExportAsync(new DatabaseSmugglerExportOptions(), exportFile);
+            await exportOperation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+
+            var operation = await dstStore.Smuggler.ImportAsync(new DatabaseSmugglerImportOptions(), exportFile);
+            await operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+
+            var destinationRecord =
+                await dstStore.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(dstStore.Database));
+            Assert.Equal(1, destinationRecord.QueueConnectionStrings.Count);
+            Assert.Equal(1, destinationRecord.QueueEtls.Count);
+
+            Assert.Equal(QueueBrokerType.AzureQueueStorage, destinationRecord.QueueEtls[0].BrokerType);
+            Assert.Equal(DefaultScript, destinationRecord.QueueEtls[0].Transforms[0].Script);
+            Assert.Equal(DefaultCollections, destinationRecord.QueueEtls[0].Transforms[0].Collections);
+
+            Assert.Equal(1, destinationRecord.QueueEtls[0].Queues.Count);
+            Assert.Equal("Orders", destinationRecord.QueueEtls[0].Queues[0].Name);
+            Assert.True(destinationRecord.QueueEtls[0].Queues[0].DeleteProcessedDocuments);
+        }
+    }
+    
+    [RavenFact(RavenTestCategory.BackupExportImport | RavenTestCategory.Sharding | RavenTestCategory.Etl)]
+    public async Task ShouldSkipUnsupportedFeaturesInShardingOnImport_RabbitMqEtl()
+    {
+        using (var srcStore = GetDocumentStore())
+        using (var dstStore = Sharding.GetDocumentStore())
+        {
+            var config = SetupQueueEtlToAzureQueueStorageOnline(srcStore,
+                DefaultScript, DefaultCollections, new List<EtlQueue>()
+                {
+                    new()
+                    {
+                        Name = "Orders",
+                        DeleteProcessedDocuments = true
+                    }
+                }, connectionString: ConnectionString);
+
+            var record = await srcStore.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(srcStore.Database));
+
+            Assert.NotNull(record.QueueEtls);
+            Assert.Equal(1, record.QueueEtls.Count);
+            
+            var exportFile = GetTempFileName();
+
+            var exportOperation = await srcStore.Smuggler.ExportAsync(new DatabaseSmugglerExportOptions(), exportFile);
+            await exportOperation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+
+            var operation = await dstStore.Smuggler.ImportAsync(new DatabaseSmugglerImportOptions(), exportFile);
+            await operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+
+            record = await dstStore.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(dstStore.Database));
+
+            Assert.Empty(record.QueueEtls);
+        }
+    }
```

Comment:
Can we have tests which intentionally try to push too big messages?

Reply:
done